### PR TITLE
Pd 70 use variables for source databases

### PR DESCRIPTION
**Pull request by bpruss**
This update allow us to put variables in the project and read them.  
Readme should explain. If not reject this PR.

### CODE REVIEW

File: dbt_project.yml [hunk 1]
```diff
@@ -4,18 +4,20 @@
...
 # */
-
-
-config-version: 2
-
 name: 'cdc_dbt_codegen'
-version: '0.1.4'
-
-
+version: '0.1.5'
```
Comment (eskamarock) [inline dbt_project.yml]: Change to 0.2.0

File: dbt_project.yml [hunk 2]
```diff
@@ -4,18 +4,20 @@
...
+config-version: 2
 require-dbt-version: [">=1.0.0", "<2.0.0"]
 
+vars:
+  cdc_dbt_codegen:
```
Comment (eskamarock) [inline dbt_project.yml]: Specify local or make global?

File: dbt_project.yml [hunk 3]
```diff
@@ -4,18 +4,20 @@
...
+    source_db: ['snowflake']
+
 models:
-  dw_util:
-    materialized: table    
+  cdc_dbt_codegen:
```
Comment (eskamarock) [inline dbt_project.yml:16]: Change necessary?

**Review by eskamarock — CHANGES_REQUESTED** [review]
A couple of comments need addressing before approving the PR.

File: dbt_project.yml (hunk 2 above)
Comment (eskamarock) [reply dbt_project.yml]: In the readme.md, this is communicated as global vars, not specific to cdc_dbt_codegen

**Comment by bpruss** [comment]
@eskamarock - I think I've covered requested changes.

**Review by eskamarock — APPROVED** [review]
LGTM

**Pull request merged**